misc/snapshotter/entrypoint.sh

```diff
@@ -1,19 +1,24 @@
 #! /bin/bash
 
-CONTAINERD_ROOT="${CONTAINERD_ROOT:-/var/lib/containerd/}"
+NYDUS_LIB="${NYDUS_LIB:-/var/lib/containerd-nydus}"
+NYDUS_RUN="${NYDUS_RUN:-/run/containerd-nydus}"
+ENABLE_NYDUS_OVERLAY="${ENABLE_NYDUS_OVERLAY:-true}"
+ENABLE_METRICS="${ENABLE_METRICS:-false}"
+LEVEL="${LEVEL:-info}"
 
 set -eu
 BACKEND_TYPE="${BACKEND_TYPE:-config}"
 NYDUSD_DAEMON_MODE="${NYDUSD_DAEMON_MODE:-shared}"
```

Comment:
Not relating to this PR, share we change the daemon mode from `shared` to `multiple`?

Reply:
@imeoer your option?